libs/libarchfpga/src/physical_types.h

```diff
@@ -576,6 +576,23 @@ constexpr int DEFAULT_SWITCH = -2;
  * num_receivers: Total number of input receivers supplied
  * index: Keep track of type in array for easy access
  * logical_tile_index: index of the corresponding logical block type
+ *
+ * In general, the physical tile is a placeable physical resource on the FPGA device,
+ * and it is allowed to contain an heterogeneous set of logical blocks (pb_types).
+ *
+ * Each physical tile must specify at least one sub tile, that is a physical location
+ * on the z axis. This means that a physical tile occupies an (x, y) location on the grid,
+ * and it has at least one sub tile slot that allows for a placement within the (x, y) location.
+ *
+ * Therefore, to identify the location of a logical block within the device grid, we need to
+ * specify three different coordinates:
+ *      - x: horizontal coordinate
+ *      - y: vertical coordinate
+ *      - z: depth coordinate, which refers to the sub tile instance that is used within the (x, y) location
```

Comment:
rename to sub_tile index, within the sub_tile stack.

Reply:
Done